app/src/examples/RuntimeTests/tests/layoutAnimations/entering/predefinedEntering.test.tsx

```diff
@@ -0,0 +1,166 @@
+import { View, StyleSheet } from 'react-native';
+import Animated, {
+  FadeIn,
+  FadeInRight,
+  FadeInLeft,
+  FadeInUp,
+  FadeInDown,
+  BounceIn,
+  BounceInRight,
+  BounceInLeft,
+  BounceInUp,
+  BounceInDown,
+  FlipInEasyX,
+  FlipInEasyY,
+  FlipInXDown,
+  FlipInXUp,
+  FlipInYLeft,
+  FlipInYRight,
+  LightSpeedInRight,
+  LightSpeedInLeft,
+  PinwheelIn,
+  RotateInDownLeft,
+  RotateInDownRight,
+  RotateInUpLeft,
+  RotateInUpRight,
+  SlideInRight,
+  SlideInLeft,
+  SlideInUp,
+  SlideInDown,
+  StretchInX,
+  StretchInY,
+  ZoomIn,
+  ZoomInDown,
+  ZoomInEasyDown,
+  ZoomInEasyUp,
+  ZoomInLeft,
+  ZoomInRight,
+  ZoomInRotate,
+  ZoomInUp,
+} from 'react-native-reanimated';
+import React from 'react';
+import {
+  describe,
+  test,
+  expect,
+  mockAnimationTimer,
+  recordAnimationUpdates,
+  render,
+  wait,
+  unmockAnimationTimer,
+  clearRenderOutput,
+} from '../../../ReanimatedRuntimeTestsRunner/RuntimeTestsApi';
+import { PredefinedEnteringSnapshots as Snapshots } from './entering.snapshot';
+
+const FADE_ENTERING = [FadeIn, FadeInRight, FadeInLeft, FadeInUp, FadeInDown];
+const BOUNCE_ENTERING = [BounceIn, BounceInRight, BounceInLeft, BounceInUp, BounceInDown];
+const FLIP_ENTERING = [FlipInEasyX, FlipInEasyY, FlipInXDown, FlipInXUp, FlipInYLeft, FlipInYRight];
+const LIGHTSPEED_ENTERING = [LightSpeedInRight, LightSpeedInLeft];
+const PINWHEEL_ENTERING = [PinwheelIn];
+// const ROLL_ENTERING = [RollInRight, RollInLeft]; Don't test RollIn, until recording rotation snapshot gets fixed
+const ROTATE_ENTERING = [RotateInDownLeft, RotateInDownRight, RotateInUpLeft, RotateInUpRight];
+const SLIDE_ENTERING = [SlideInRight, SlideInLeft, SlideInUp, SlideInDown];
+const STRETCH_ENTERING = [StretchInX, StretchInY];
+
+const ZOOM_ENTERING = [
+  ZoomIn,
+  ZoomInDown,
+  ZoomInEasyDown,
+  ZoomInEasyUp,
+  ZoomInLeft,
+  ZoomInRight,
+  ZoomInRotate,
+  ZoomInUp,
+];
+
+const ENTERING_SETS: Array<[string, unknown[], number]> = [
```

Comment:
Is there any advantage of `unknown[]` over `any[]`? Just curious since later on we treat each Entering class as `any`.

Reply:
Not really, however its considered to be a good custom to always use `unknown` instead of `any` if possible.